Stellar-ledger-entries.x

```diff
@@ -643,19 +621,22 @@ case CONTRACT_DATA:
         SCAddress contract;
         SCVal key;
         ContractDataDurability durability;
-        ContractEntryBodyType bodyType;
     } contractData;
 case CONTRACT_CODE:
     struct
     {
         Hash hash;
-        ContractEntryBodyType bodyType;
     } contractCode;
 case CONFIG_SETTING:
     struct
     {
         ConfigSettingID configSettingID;
     } configSetting;
+case EXPIRATION:
+    struct
+    {
+        Hash keyHash;
```

Comment:
do you need a corresponding preimage variant, e.g. `HashIdPreimageLedgerKey`? not sure if you guys always have a 1:1 relationship between a hash property and a corresponding preimage structure

Reply:
It can be just hash of `LedgerKey`. Preimages are needed for external stuff, but for internal stuff I don't think we need to distinguish hash namespaces.

Reply:
Fine by me - a comment here would be appreciated, then :+1: